core_backend/app/schemas.py

```diff
@@ -44,8 +44,8 @@ class UserQuerySearchResult(BaseModel):
     Pydantic model for each individual search result
     """
 
-    response_title: Optional[str] = ""  # TODO: optional or not?
-    response_text: str
+    retrieved_title: str
```

Comment:
makes a lot more sense!

Reply:
Maybe we can change `embiddings-search` also into `/retrieve` or something, to be more consistent?

Reply:
Yeah - I don't like `embeddings-search` though I think I came up with it lol.

Reply:
Some day..